relnotes/resume.feature.md

```diff
@@ -0,0 +1,13 @@
+## Delayed resuming of a task
+
+`ocean.task.IScheduler`
+
+`IScheduler` interface defines new method, `delayedResume`, which takes one task
+and adds it to the same resume queue as used by `processEvents` method. That
+means that such task will be resumed by the scheduler after current epoll cycle
+instead of immediate resuming by the current context.
```

Comment:
Is it worth explicitly mentioning that the task will be resumed _from_ the epoll context, too? (Also in method comments.)

Reply:
Isn't it already implied by "same resume queue as used by `processEvents`" and "resumed by the scheduler"?

Reply:
Ah, "resumed by the scheduler", yes.

I'd remove the bit about `processEvents`, tbh. That's an implementation detail and irrelevant to the average user. That's what threw me off.

Reply:
Updated